pkg/dockerregistry/server/blobdescriptorservice.go

```diff
@@ -25,6 +38,155 @@ type blobDescriptorService struct {
 	distribution.BlobDescriptorService
 }
 
+// Stat returns a a blob descriptor if the given blob is either linked in repository or is referenced in
+// corresponding image stream. This method is invoked from inside of upstream's linkedBlobStore. It expects
```

Comment:
Yes, it is. It just doesn't apply to a cross-repo mount stat in the source repository anymore. All the other stat calls go through this service.